vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -0,0 +1,210 @@
+        >
+          Do we have a team with the right skills?
+        </div>
+      </div>
+      <div
+        class="tw-mt-[1.125rem] tw-max-w-[23.313rem] md:tw-max-w-[50.688rem]"
+      >
+        <div
+          class="tw-w-full tw-text-center tw-font-inter-medium tw-text-black-core/[0.6] tw-text-[1rem] md:tw-text-[1.25rem] xl:tw-text-[1.5rem] tw-leading-[1.375rem] md:tw-leading-[2rem] xl:tw-leading-[2.625rem]"
+        >
+          Well, at least the community says Hell Yeah. Our blogs hosted on
+          medium have
+          <span class="v2-canopas-gradient-text tw-font-inter-bold">100k+</span>
+          minutes monthly reading time and it’s only rising.
+        </div>
+      </div>
+    </div>
+    <div
+      class="tw-relative tw-flex tw-flex-col tw-items-center md:tw-items-start tw-mt-[6.813rem]"
+    >
+      <img
+        v-if="width < 680"
+        :src="bgMobile"
+        loading="lazy"
+        :class="blogs.length == 2 ? '-tw-top-[2rem]' : '-tw-top-[3rem]'"
+        class="tw-absolute tw-right-0"
+        alt="blog-background-image"
+      />
+      <img
+        v-else
+        :src="bg"
+        :class="blogs.length == 2 ? '-tw-top-[2rem]' : 'tw-top-0'"
+        class="tw-absolute tw-right-0"
+        alt="blog-background-image"
+      />
+
+      <div class="md:tw-mt-[4.625rem] tw-z-[1]">
+        <hr
+          class="tw-hidden md:tw-block tw-mt-[1.125rem] md:tw-ml-[4.375rem] lg:tw-ml-[8.25rem] 2xl:tw-ml-[9.313rem] md:tw-w-[18.438rem] lg:tw-w-[27.5rem] 2xl:tw-w-[42.375rem] tw-h-[0.063] tw-bg-[#C6C6C6]"
+        />
+        <div
+          v-for="(blog, index) in blogs"
+          :key="blog"
+          @mouseover="activeIndex = index"
+          @mouseleave="activeIndex = index"
+          @touchstart.passive="activeIndex = index"
+          @click="openLink"
+          class="tw-container lg:tw-ml-[48px]"
+          @click.native="$gtag.event('tap_blog_post')"
+        >
+          <img
+            v-show="activeIndex == index"
+            :src="blog.thumbnail"
+            :class="
+              blogs.length == 2
+                ? 'tw-top-[11.76rem] 2xl:tw-top-[8.625rem]'
+                : 'tw-top-[14.2rem] lg:tw-top-[13.3rem] 2xl:tw-top-[10.625rem]'
+            "
+            class="tw-hidden md:tw-block tw-absolute tw-right-10 tw-w-[20rem] 2xl:tw-w-[31.875rem] tw-max-h-[13.75rem] 2xl:tw-max-h-[20rem] tw-animate-fadeIn"
+            loading="lazy"
+            :alt="blog.title"
+          />
+
+          <div class="tw-flex tw-flex-row tw-gap-8">
+            <div class="tw-flex tw-flex-col tw-pt-[1.5rem] md:tw-pt-[2.120rem]">
+              <div
+                class="tw-text-black-core/[.87] tw-font-inter-semibold md:tw-font-inter-regular tw-text-[1.5rem] md:tw-text-[2.188rem] 2xl:tw-text-[3.125rem] tw-leading-[1.95rem] md:tw-leading-[2.183rem] 2xl:tw-leading-[4.063rem]"
```

Comment:
Let's put `tw-text-black-core/[.87] `  in the first div of this section as it used in the whole component